pkg/models/model_helpers.go

```diff
@@ -16,3 +16,11 @@ func HTMLPointer(input string) *HTML {
 	html := HTML(input)
 	return &html
 }
+
+// ValueOrEmpty returns a string if the input is not nil, otherwise returns an empty string
+func ValueOrEmpty(st *string) string {
```

Comment:
Is this being used anywhere?

Reply:
good catch, thanks! It was being used, but I updated the username field in the UserAccount table to not be a pointer. We can add this again if needed.